webpack.config.js

```diff
@@ -206,11 +206,11 @@ function getWebpackConfig( env, argv ) {
 			// Handle SVG images only in CSS files.
 			return {
 				...rule,
-				test: /\.(?:gif|jpg|jpeg|png|woff|woff2|eot|ttf|otf|svg)$/i,
 				issuer: styleSheetFiles,
+				type: 'asset/resource',
```

Comment:
By default `@wordpress/scripts` is adding it as `type: 'asset/inline',` for style files.

Reply:
@renatho could you help me out with this? I've managed to break it when updating the packages and can't figure out why the inline SVGs are causing an error:

```
ERROR in ../node_modules/@automattic/data-stores/node_modules/@automattic/calypso-products/node_modules/@automattic/components/src/icons/sparkles.tsx 5:1
Module parse failed: Unexpected token (5:1)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.
```

Reply:
Hi @m1r0! I'll take a look and see if I can find the cause on Monday. 😉

Reply:
I've managed to solve the above by updating `@automattic/components` and `@automattic/tour-kit` (related PRs https://github.com/Automattic/wp-calypso/pull/90267 https://github.com/woocommerce/woocommerce/pull/47129) but now the test suite is throwing `Cannot unlock an object that was not locked before`. 🙄 Are you familiar with this error?

Reply:
I found that this is an error coming from Gutenberg: https://github.com/wordpress/gutenberg/commit/1b1810456daf24439fa4745165855d3f4592ecb1#diff-a115d786e2db4593eaf8ecdf7126885b1fb7f2c936c2884de7f52a5ab00eb1e0R142-R144

But no idea of what it means without investigating deeper. :(